CDCgov/prime-reportstream
# Engagement/angela/8738/optional batch headers

This PR ...

Makes Batch Headers optional for HL7 output files by making use of the useBatchHeaders field in a receiver's translation settings.

Test Steps:
1. Set up a receiver with useBatchHeaders set to false in their translation settings.
2. Create a file that can be run through the UP for that receiver.
3. Run file through UP.
4. Verify file does not have batch headers.
5. Set useBatchHeaders to true.
6. Run file through UP.
7. Verify file does have batch headers.

## Changes
- Added a checks for useBatchHeaders to the batchMessages function in HL7MessageHelpers
- Added a unit test to make sure that when a Receiver had the setting set to false, batch headers were not being added

## Checklist

### Testing

## Linked Issues
- Fixes #8738
